compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -1419,9 +1375,4 @@ impl Builder<'a, 'll, 'tcx> {
             llvm::LLVMAddIncoming(phi, &val, &bb, 1 as c_uint);
         }
     }
-
-    fn wasm_and_missing_nontrapping_fptoint(&self) -> bool {
-        self.sess().target.arch == "wasm32"
-            && !self.sess().target_features.contains(&sym::nontrapping_dash_fptoint)
```

Comment:
`nontrapping_dash_fptoint` should be removed from a list of symbols now that it no longer has any uses remaining.